harmonic/chains.pyx

```diff
@@ -485,7 +485,7 @@ class Chains:
             # print("chain = {}, start_indices_new = {}"
             #.format(i_chain, start_indices_new))
         
-        self.start_indices = start_indices_new
+        self.start_indices = start_indices_new.tolist()
```

Comment:
Ensures consistency in the way code is called elsewhere. Indices should be lists throughout.